cli/commands/data_restore.go

```diff
@@ -37,8 +38,12 @@ func (cmd *DataRestore) Commands() []cli.Command {
 }
 
 func (cmd *DataRestore) Run(c *cli.Context) error {
+	if runtime.GOOS == "linux" {
+		return cmd.Success("Data Restore is not needed on Linux, please unarchive any data directly")
```

Comment:
This makes sense to me, but if we ever expanded on the data backup to include volume wrangling this would change.